packages/client/utils/constants.ts

```diff
@@ -116,11 +121,6 @@ export const GITHUB_ENDPOINT = 'https://api.github.com/graphql'
 /* JavaScript specifics */
 export const MAX_INT = 2147483647
 
-/* Relay Subscription Channels */
-export const NOTIFICATION = 'notification'
-export const ORGANIZATION = 'organization'
```

Comment:
No instances of these two constants were used.  `SubscriptionChannel.NOTIFICATION|ORGANIZATION` are used instead.